src/main/webapp/i18n/en/programmingExercise.json

```diff
@@ -88,7 +88,14 @@
                 "testCasesCouldNotBeUpdated": "The following test cases could not be updated: {{testCases}}."
             },
             "resubmitOnFailedSubmission": "The result for the last submission of this participation could not be generated. Click the button to submit again. This will trigger a new build run that tests the last submission.",
-            "resubmit": "Resubmit the last submission. This will execute a build run on the participation's repository without making a commit."
+            "resubmit": "Resubmit the last submission. This will execute a build run on the participation's repository without making a commit.",
+            "resubmitAll": "Trigger all",
+            "resubmitAllTooltip": "Trigger the last submission of all participations again. The results will be rated.",
+            "resubmitAllDialog": "WARNING: Triggering all participations again is a very expensive operation. This action will start a CI build for every participation in this exercise!",
```

Comment:
Please something like "This means, that all results of the students will be calculated again and will be overridden!"

Reply:
Done!